Cargo.toml

```diff
@@ -71,7 +71,7 @@ keyframe = "1"
 keyframe_derive = "1"
 num-derive = "0.4"
 
-skeptic = "0.13"
+skeptic = { git = "https://github.com/Erk-/rust-skeptic.git", branch = "fix-rust-1-77" }
```

Comment:
@a1phyr This is a temp fix so we can keep doctests running in CI until I can find an alternative for `skeptic`.

I cherry-picked your other CI fixes here to test everything together.